add cloud cli video and alias info

mirnawong1 opened the pull request:
this pr adds steve's cloud-cli video instructions and also adds info about aliasing cloud cli for dbt core users

Commit:
adding steve's video

matthewshaver commented on website/docs/docs/cloud/cloud-cli-installation.md:
```diff
@@ -29,7 +29,7 @@ You must be on dbt version 1.5 or higher. Refer to [dbt Cloud versions](/docs/db
 ## Install dbt Cloud CLI
 
-You can install the dbt Cloud CLI on the command line by using one of these methods:
+You can install the dbt Cloud CLI on the command line by using one of these methods. For a video instruction step-by-step guide, refer to the [FAQs](#faqs).
```
```suggestion
You can install the dbt Cloud CLI on the command line by using one of these methods. For a step-by-step video guide, refer to the [FAQs](#faqs).
```

mirnawong1 committed:
Update website/docs/docs/cloud/cloud-cli-installation.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

mirnawong1 committed:
Merge branch 'current' into mwong-add-cloud-cli-loom

matthewshaver commented on website/docs/docs/cloud/cloud-cli-installation.md:
````diff
@@ -38,24 +38,35 @@ You can install the dbt Cloud CLI on the command line by using one of these meth
-  ```
-
-3. Verify the installation by running `dbt --help` from the command line. If the help text doesn't indicate that you're using the dbt Cloud CLI, make sure you've deactivated your pyenv or venv and don't have a version of dbt globally installed.
+   - First, remove a tap, which is like a separate repository for packages, from Homebrew. This stops Homebrew from installing packages from that repository:
````
Adding specificity might help users following the instructions. 
```suggestion
   - First, remove the dbt-labs tap, the separate repository for packages, from Homebrew. This prevents Homebrew from installing packages from that repository:
```

mirnawong1 committed:
Update website/docs/docs/cloud/cloud-cli-installation.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

matthewshaver commented on website/docs/docs/cloud/cloud-cli-installation.md:
````diff
@@ -38,24 +38,35 @@ You can install the dbt Cloud CLI on the command line by using one of these meth
+      brew install dbt
+      ```
+
+3. Verify the installation by running `dbt --help` from the command line. 
````
Should we drop some sample text of what the expected output is?

matthewshaver commented on website/docs/docs/cloud/cloud-cli-installation.md:
````diff
@@ -38,24 +38,35 @@ You can install the dbt Cloud CLI on the command line by using one of these meth
+      ```
+
+3. Verify the installation by running `dbt --help` from the command line. 
+   * If the help text doesn't indicate that you're using the dbt Cloud CLI, make sure you've deactivated your pyenv or venv and don't have a global dbt version installed.
````
Should we drop some sample text of what the expected output is?

mirnawong1 replied:
i thought about it and opted to leave it out because `dbt --help` returns a pretty long output (basically it gives users all the flags/commands they can use) so i decided to leave it out and thought that the 'help text...' would be clear enough

matthewshaver replied:
"`dbt --help` returns a pretty long output.." is the text that indicates it is successfully running for cloud going to be immediately apparent in this output or will they have to dig for it? Maybe including just a sample of what to look for so they know 100% it was successful?

matthewshaver replied:
I think I'm maybe just confused by what this sentence is saying. Are we saying that the ability to run `dbt --help` is enough to confirm installation, or are they supposed to see something that specifically states it's the Cloud CLI and not the Core CLI?  If it's the former, maybe we can clarify it by saying something like: "If the installation was successful, executing `dbt --help` will produce a list of commands available to run."

mirnawong1 replied:
ok maybe this clarifies it further?

![Screenshot 2023-10-25 at 10 39 43](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/7f3f76e9-5b00-421f-92ae-7394cc78e86f)

mirnawong1 commented:
this is the `dbt --help` text btw @matthewshaver 

```
Usage: dbt run [OPTIONS]

  Compile SQL and execute against the current target database.

Options:
  --defer / --no-defer            If set, resolve unselected nodes by
                                  deferring to the manifest within the --state
                                  directory.
  --favor-state / --no-favor-state
                                  If set, defer to the argument provided to
                                  the state flag for resolving unselected
                                  nodes, even if the node(s) exist as a
                                  database object in the current environment.
  --deprecated-favor-state TEXT   Internal flag for deprecating old env var.
  --exclude TUPLE                 Specify the nodes to exclude.
  -x, --fail-fast / --no-fail-fast
                                  Stop execution on first failure.
  -f, --full-refresh              If specified, dbt will drop incremental
                                  models and fully-recalculate the incremental
                                  table from the model definition.
  --profile TEXT                  Which profile to load. Overrides setting in
                                  dbt_project.yml.
  --profiles-dir PATH             Which directory to look in for the
                                  profiles.yml file. If not set, dbt will look
                                  in the current working directory first, then
                                  HOME/.dbt/
  --project-dir PATH              Which directory to look in for the
                                  dbt_project.yml file. Default is the current
                                  working directory and its parents.
  -s, -m, --select, --models, --model TUPLE
                                  Specify the nodes to include.
  --selector TEXT                 The selector name to use, as defined in
                                  selectors.yml
  --state DIRECTORY               Unless overridden, use this state directory
                                  for both state comparison and deferral.
  --defer-state DIRECTORY         Override the state directory for deferral
                                  only.
  -t, --target TEXT               Which target to load for the given profile
  --target-path PATH              Configure the 'target-path'. Only applies
                                  this setting for the current run. Overrides
                                  the 'DBT_TARGET_PATH' if it is set.
  --threads INTEGER               Specify number of threads to use while
                                  executing models. Overrides settings in
                                  profiles.yml.
  --vars YAML                     Supply variables to the project. This
                                  argument overrides variables defined in your
                                  dbt_project.yml file. This argument should
                                  be a YAML string, eg. '{my_variable:
                                  my_value}'
  --version-check / --no-version-check
                                  If set, ensure the installed dbt version
                                  matches the require-dbt-version specified in
                                  the dbt_project.yml file (if any).
                                  Otherwise, allow them to differ.
  -h, --help                      Show this message and exit.
➜  dbt_zero git:(current) ✗ dbt run --select dim_hosts_clean.sql
Usage: dbt run [OPTIONS]
Try 'dbt run -h' for help.

Error: Invalid value for '--profiles-dir': Path '/Users/mirnawong/.dbt' does not exist.
➜  dbt_zero git:(current) ✗ ls
README.md       dbt_cloud.yml   dbt_project.yml macros          packages.yml    snapshots       tests
analyses        dbt_packages    logs            models          seeds           target
➜  dbt_zero git:(current) ✗ cd ..
➜  dbt-projects git:(current) ✗ ls
dbt_zero
➜  dbt-projects git:(current) ✗ --version-checjk
zsh: command not found: --version-checjk
➜  dbt-projects git:(current) ✗ --version-check
zsh: command not found: --version-check
➜  dbt-projects git:(current) ✗ dbt --version-check
➜  dbt-projects git:(current) ✗ dbt compile
Usage: dbt compile [OPTIONS]
Try 'dbt compile -h' for help.

Error: Invalid value for '--profiles-dir': Path '/Users/mirnawong/.dbt' does not exist.
➜  dbt-projects git:(current) ✗ dbt --project-dir /
Session Contents Restored on 12 Sep 2023 at 09:14
Last login: Tue Sep 12 09:13:42 on console
➜  dbt-projects git:(current) ✗
Session Contents Restored on 13 Sep 2023 at 08:13
Last login: Wed Sep 13 08:13:08 on console
➜  dbt-projects git:(current) ✗
Session Contents Restored on 14 Sep 2023 at 08:40
Last login: Thu Sep 14 08:40:22 on console
➜  dbt-projects git:(current) ✗
Session Contents Restored on 15 Sep 2023 at 08:43
Last login: Fri Sep 15 08:43:34 on console
➜  dbt-projects git:(current) ✗
Session Contents Restored on 16 Sep 2023 at 12:06
Last login: Sat Sep 16 12:05:59 on console
➜  dbt-projects git:(current) ✗
Session Contents Restored on 18 Sep 2023 at 08:50
Last login: Mon Sep 18 08:50:18 on console
➜  dbt-projects git:(current) ✗
Session Contents Restored on 19 Sep 2023 at 08:51
Last login: Tue Sep 19 08:51:31 on console
➜  dbt-projects git:(current) ✗
Session Contents Restored on 19 Sep 2023 at 15:11
Last login: Tue Sep 19 15:11:35 on console
➜  dbt-projects git:(current) ✗
Session Contents Restored on 19 Sep 2023 at 19:14
Last login: Tue Sep 19 19:14:37 on console
➜  dbt-projects git:(current) ✗
Session Contents Restored on 21 Sep 2023 at 17:20
Last login: Thu Sep 21 17:20:04 on console
➜  dbt-projects git:(current) ✗
Session Contents Restored on 22 Sep 2023 at 12:54
Last login: Fri Sep 22 12:54:11 on console
➜  dbt-projects git:(current) ✗
Session Contents Restored on 25 Sep 2023 at 08:21
Last login: Mon Sep 25 08:21:07 on console
[oh-my-zsh] Would you like to update? [Y/n]
Session Contents Restored on 27 Sep 2023 at 08:20
Last login: Wed Sep 27 08:20:20 on console
[oh-my-zsh] Would you like to update? [Y/n]
Session Contents Restored on 28 Sep 2023 at 08:38
Last login: Thu Sep 28 08:37:49 on console
[oh-my-zsh] Would you like to update? [Y/n]
Session Contents Restored on 29 Sep 2023 at 08:41
Last login: Fri Sep 29 08:41:06 on console
[oh-my-zsh] Would you like to update? [Y/n]
Session Contents Restored on 2 Oct 2023 at 09:30
Last login: Mon Oct  2 09:29:46 on console
➜  dbt-projects git:(current) ✗
Session Contents Restored on 3 Oct 2023 at 09:14
Last login: Tue Oct  3 09:14:39 on console
➜  dbt-projects git:(current) ✗
Session Contents Restored on 4 Oct 2023 at 08:34
Last login: Wed Oct  4 08:34:24 on console
➜  dbt-projects git:(current) ✗
Session Contents Restored on 5 Oct 2023 at 08:37
Last login: Thu Oct  5 08:37:08 on console
➜  dbt-projects git:(current) ✗
Session Contents Restored on 24 Oct 2023 at 16:03
Last login: Tue Oct 24 08:48:10 on console
➜  dbt-projects git:(current) ✗ ls
dbt_zero     dbt_zero.zip
➜  dbt-projects git:(current) ✗ cd dbt_zero
➜  dbt_zero git:(current) ✗ ls
README.md       dbt_cloud.yml   dbt_project.yml macros          packages.yml    snapshots       tests
analyses        dbt_packages    logs            models          seeds           target
➜  dbt_zero git:(current) ✗ dbt --help
Usage: dbt [OPTIONS] COMMAND [ARGS]...

  An ELT tool for managing your SQL transformations and data models. For more
  documentation on these commands, visit: docs.getdbt.com

Options:
  --cache-selected-only / --no-cache-selected-only
                                  At start of run, populate relational cache
                                  only for schemas containing selected nodes,
                                  or for all schemas of interest.
  -d, --debug / --no-debug        Display debug logging during dbt execution.
                                  Useful for debugging and making bug reports.
  -x, --fail-fast / --no-fail-fast
                                  Stop execution on first failure.
  --log-cache-events / --no-log-cache-events
                                  Enable verbose logging for relational cache
                                  events to help when debugging.
  --log-format [text|debug|json|default]
                                  Specify the format of logging to the console
                                  and the log file. Use --log-format-file to
                                  configure the format for the log file
                                  differently than the console.
  --log-format-file [text|debug|json|default]
                                  Specify the format of logging to the log
                                  file by overriding the default value and the
                                  general --log-format setting.
  --log-level [debug|info|warn|error|none]
                                  Specify the minimum severity of events that
                                  are logged to the console and the log file.
                                  Use --log-level-file to configure the
                                  severity for the log file differently than
                                  the console.
  --log-level-file [debug|info|warn|error|none]
                                  Specify the minimum severity of events that
                                  are logged to the log file by overriding the
                                  default value and the general --log-level
                                  setting.
  --log-path PATH                 Configure the 'log-path'. Only applies this
                                  setting for the current run. Overrides the
                                  'DBT_LOG_PATH' if it is set.
  --partial-parse / --no-partial-parse
                                  Allow for partial parsing by looking for and
                                  writing to a pickle file in the target
                                  directory. This overrides the user
                                  configuration file.
  --populate-cache / --no-populate-cache
                                  At start of run, use `show` or
                                  `information_schema` queries to populate a
                                  relational cache, which can speed up
                                  subsequent materializations.
  --print / --no-print            Output all {{ print() }} macro calls.
  --printer-width INTEGER         Sets the width of terminal output
  -q, --quiet / --no-quiet        Suppress all non-error logging to stdout.
                                  Does not affect {{ print() }} macro calls.
  -r, --record-timing-info PATH   When this option is passed, dbt will output
                                  low-level timing stats to the specified
                                  file. Example: `--record-timing-info
                                  output.profile`
  --send-anonymous-usage-stats / --no-send-anonymous-usage-stats
                                  Send anonymous usage stats to dbt Labs.
  --static-parser / --no-static-parser
                                  Use the static parser.
  --use-colors / --no-use-colors  Specify whether log output is colorized in
                                  the console and the log file. Use --use-
                                  colors-file/--no-use-colors-file to colorize
                                  the log file differently than the console.
  --use-colors-file / --no-use-colors-file
                                  Specify whether log file output is colorized
                                  by overriding the default value and the
                                  general --use-colors/--no-use-colors
                                  setting.
  --use-experimental-parser / --no-use-experimental-parser
                                  Enable experimental parsing features.
  -V, -v, --version               Show version information and exit
  --version-check / --no-version-check
                                  If set, ensure the installed dbt version
                                  matches the require-dbt-version specified in
                                  the dbt_project.yml file (if any).
                                  Otherwise, allow them to differ.
  --warn-error                    If dbt would normally warn, instead raise an
                                  exception. Examples include --select that
                                  selects nothing, deprecations,
                                  configurations with no associated models,
                                  invalid test configurations, and missing
                                  sources/refs in tests.
  --warn-error-options WARNERROROPTIONSTYPE
                                  If dbt would normally warn, instead raise an
                                  exception based on include/exclude
                                  configuration. Examples include --select
                                  that selects nothing, deprecations,
                                  configurations with no associated models,
                                  invalid test configurations, and missing
                                  sources/refs in tests. This argument should
                                  be a YAML string, with keys 'include' or
                                  'exclude'. eg. '{"include": "all",
                                  "exclude": ["NoNodesForSelectionCriteria"]}'
  --write-json / --no-write-json  Whether or not to write the manifest.json
                                  and run_results.json files to the target
                                  directory
  -h, --help                      Show this message and exit.

Commands:
  build          Run all seeds, models, snapshots, and tests in DAG order
  clean          Delete all folders in the clean-targets list (usually...
  clone          Create clones of selected nodes based on their location...
  compile        Generates executable SQL from source, model, test, and...
  debug          Show information on the current dbt environment and...
  deps           Pull the most recent version of the dependencies listed...
  docs           Generate or serve the documentation website for your...
  init           Initialize a new dbt project.
  list           List the resources in your project
  parse          Parses the project and provides information on performance
  retry          Retry the nodes that failed in the previous run.
  run            Compile SQL and execute against the current target...
  run-operation  Run the named macro with any supplied arguments.
  seed           Load data from csv files into your data warehouse.
  show           Generates executable SQL for a named resource or inline...
  snapshot       Execute snapshots defined in your project
  source         Manage your project's sources
  test           Runs tests on data in deployed models.

  Specify one of these sub-commands and you can find more help from there.
```

matthewshaver commented on website/docs/docs/cloud/cloud-cli-installation.md:
```diff
@@ -38,24 +38,35 @@ You can install the dbt Cloud CLI on the command line by using one of these meth
+   * If the help text doesn't indicate that you're using the dbt Cloud CLI, make sure you've deactivated your pyenv or venv and don't have a global dbt version installed.
    * You don't have to run the `dbt deps` command when your environment starts. Previously, you had to do it during initialization. However, you'll still need to run `dbt deps` if you make changes to your `packages.yml` file.
 
+4. After installation, [configure](/docs/cloud/configure-cloud-cli) the dbt Cloud CLI for your dbt Cloud project and use it to run [dbt commands](/reference/dbt-commands) similar to dbt Core. For example, execute `dbt compile`, to compile a project using dbt Cloud and confirm that it works.
```
Might be helpful to give an idea of what `dbt compile` does in the instructions
```suggestion
4. After you've verified the installation, [configure](/docs/cloud/configure-cloud-cli) the dbt Cloud CLI for your dbt Cloud project and use it to run [dbt commands](/reference/dbt-commands) similar to dbt Core. For example, execute `dbt compile` to compile a project using dbt Cloud and validate your models and tests.
```

mirnawong1 committed:
Update website/docs/docs/cloud/cloud-cli-installation.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

matthewshaver commented on website/docs/docs/cloud/cloud-cli-installation.md:
```diff
@@ -38,24 +38,35 @@ You can install the dbt Cloud CLI on the command line by using one of these meth
 
+4. After installation, [configure](/docs/cloud/configure-cloud-cli) the dbt Cloud CLI for your dbt Cloud project and use it to run [dbt commands](/reference/dbt-commands) similar to dbt Core. For example, execute `dbt compile`, to compile a project using dbt Cloud and confirm that it works.
+
+**Note**, that if you're using the dbt Cloud CLI, you can connect to your data platform directly in the dbt Cloud interface and don't need a [`profiles.yml`](/docs/core/connect-data-platform/profiles.yml) file. 
```
```suggestion
If you're using the dbt Cloud CLI, you can connect to your data platform directly in the dbt Cloud interface and don't need a [`profiles.yml`](/docs/core/connect-data-platform/profiles.yml) file locally on your machine.  
```

mirnawong1 committed:
Update website/docs/docs/cloud/cloud-cli-installation.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

matthewshaver commented on website/docs/docs/cloud/cloud-cli-installation.md:
```diff
@@ -231,3 +250,12 @@ If you have dbt Core installed locally, either:
 </details>
+
+<details>
+<summary>Are there any instructional videos to help guide me through installation? </summary>
```
I'm not opposed to this being in the FAQs, but given what we know about scroll rates, this might get more views if we embed it directly into the instructions above, rather than/in addition to putting it here.

mirnawong1 replied:
Yea i also tried adding it in the install header but then it took up too much of the page before introducing the install steps. I could link to it but that also means it'll navigate the user to another tab (and away from the docs). Which is why i figured the faq and a link to the faq was a smoother experience (as opposed to navigating them away from the docs)

matthewshaver replied:
I'll ask if we can get pop out windows for videos

matthewshaver commented on website/docs/docs/cloud/cloud-cli-installation.md:
```diff
@@ -73,9 +84,14 @@ Advanced users can configure multiple projects to use the same dbt Cloud CLI by
+   * If the help text doesn't indicate that you're using the dbt Cloud CLI, make sure you've deactivated your pyenv or venv and don't have a version of dbt globally installed.
    * You don't have to run the `dbt deps` command when your environment starts. Previously, you had to do it during initialization. However, you'll still need to run `dbt deps` if you make changes to your `packages.yml` file.
 
+4. After installation, [configure](/docs/cloud/configure-cloud-cli) the dbt Cloud CLI for your dbt Cloud project and use it to run [dbt commands](/reference/dbt-commands) similar to dbt Core. For example, execute `dbt compile`, to compile a project using dbt Cloud and confirm that it works.
```
Same suggestions as above
```suggestion
4. After you've verified the installation, [configure](/docs/cloud/configure-cloud-cli) the dbt Cloud CLI for your dbt Cloud project and use it to run [dbt commands](/reference/dbt-commands) similar to dbt Core. For example, execute `dbt compile` to compile a project using dbt Cloud and validate your models and tests.
```

matthewshaver commented on website/docs/docs/cloud/cloud-cli-installation.md:
```diff
@@ -97,23 +113,32 @@ Advanced users can configure multiple projects to use the same Cloud CLI executa
+   * If the help text doesn't indicate that you're using the dbt Cloud CLI, make sure you've deactivated your pyenv or venv and don't have a version of dbt globally installed.
    * You don't have to run the `dbt deps` command when your environment starts. Previously, you had to do it during initialization. However, you'll still need to run `dbt deps` if you make changes to your `packages.yml` file.
 
+4. After installation, [configure](/docs/cloud/configure-cloud-cli) the dbt Cloud CLI for your dbt Cloud project and use it to run [dbt commands](/reference/dbt-commands) similar to dbt Core. For example, execute `dbt compile`, to compile a project using dbt Cloud and confirm that it works.
```
```suggestion
4. After you've verified the installation, [configure](/docs/cloud/configure-cloud-cli) the dbt Cloud CLI for your dbt Cloud project and use it to run [dbt commands](/reference/dbt-commands) similar to dbt Core. For example, execute `dbt compile` to compile a project using dbt Cloud and validate your models and tests.
```

matthewshaver commented on website/docs/docs/cloud/cloud-cli-installation.md:
````diff
@@ -156,6 +181,10 @@ We recommend using virtual environments (venv) to namespace `cloud-cli`.
   pip3 install dbt-core==VERSION
   ```
 
+4. After installation, [configure](/docs/cloud/configure-cloud-cli) the dbt Cloud CLI for your dbt Cloud project and use it to run [dbt commands](/reference/dbt-commands) similar to dbt Core. For example, execute `dbt compile`, to compile a project using dbt Cloud and confirm that it works.
````
```suggestion
4. After you've verified the installation, [configure](/docs/cloud/configure-cloud-cli) the dbt Cloud CLI for your dbt Cloud project and use it to run [dbt commands](/reference/dbt-commands) similar to dbt Core. For example, execute `dbt compile` to compile a project using dbt Cloud and validate your models and tests.
```

matthewshaver commented on website/docs/docs/cloud/cloud-cli-installation.md:
```diff
@@ -156,6 +181,10 @@ We recommend using virtual environments (venv) to namespace `cloud-cli`.
 
+4. After installation, [configure](/docs/cloud/configure-cloud-cli) the dbt Cloud CLI for your dbt Cloud project and use it to run [dbt commands](/reference/dbt-commands) similar to dbt Core. For example, execute `dbt compile`, to compile a project using dbt Cloud and confirm that it works.
+
+**Note**, that if you're using the dbt Cloud CLI, you can connect to your data platform directly in the dbt Cloud interface and don't need a [`profiles.yml`](/docs/core/connect-data-platform/profiles.yml) file. 
```
```suggestion
If you're using the dbt Cloud CLI, you can connect to your data platform directly in the dbt Cloud interface and don't need a [`profiles.yml`](/docs/core/connect-data-platform/profiles.yml) file locally on your machine. 
```

matthewshaver commented on website/docs/docs/cloud/cloud-cli-installation.md:
```diff
@@ -97,23 +113,32 @@ Advanced users can configure multiple projects to use the same Cloud CLI executa
 
+4. After installation, [configure](/docs/cloud/configure-cloud-cli) the dbt Cloud CLI for your dbt Cloud project and use it to run [dbt commands](/reference/dbt-commands) similar to dbt Core. For example, execute `dbt compile`, to compile a project using dbt Cloud and confirm that it works.
+
+**Note**, that if you're using the dbt Cloud CLI, you can connect to your data platform directly in the dbt Cloud interface and don't need a [`profiles.yml`](/docs/core/connect-data-platform/profiles.yml) file. 
```
```suggestion
If you're using the dbt Cloud CLI, you can connect to your data platform directly in the dbt Cloud interface and don't need a [`profiles.yml`](/docs/core/connect-data-platform/profiles.yml) file locally on your machine. 
```

matthewshaver committed:
Merge branch 'current' into mwong-add-cloud-cli-loom

mirnawong1 committed:
Update cloud-cli-installation.md

clarify dbt --help output per Matt's feedbakc

mirnawong1 committed:
Update cloud-cli-installation.md

Commit:
rephrase

mirnawong1 committed:
Merge branch 'current' into mwong-add-cloud-cli-loom

Commit:
fold in feedback

Commit:
Merge branch 'mwong-add-cloud-cli-loom' of https://github.com/dbt-labs/docs.getdbt.com into mwong-add-cloud-cli-loom

mirnawong1 committed:
Update website/docs/docs/cloud/cloud-cli-installation.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

Commit:
add colon

Commit:
Merge branch 'mwong-add-cloud-cli-loom' of https://github.com/dbt-labs/docs.getdbt.com into mwong-add-cloud-cli-loom

mirnawong1 committed:
Merge branch 'current' into mwong-add-cloud-cli-loom

mirnawong1 commented:
thanks for checking the windows pop up question @matthewshaver , do we need to wait for front to create the component or can i merge this pr now?

mirnawong1 committed:
Update cloud-cli-installation.md

folding privatelink/ssh per [slack thread](https://dbt-labs.slack.com/archives/C051TUB7S9W/p1698164950851459?thread_ts=1698149556.814129&cid=C051TUB7S9W) in this  pr

Commit:
tweak video callout

mirnawong1 committed:
Merge branch 'current' into mwong-add-cloud-cli-loom

mirnawong1 commented on website/docs/docs/cloud/cloud-cli-installation.md:
```diff
@@ -19,42 +19,65 @@ dbt commands are run against dbt Cloud's infrastructure and benefit from:
-You must be on dbt version 1.5 or higher. Refer to [dbt Cloud versions](/docs/dbt-versions/upgrade-core-in-cloud) to upgrade.
+- Ensure you are using dbt version 1.5 or higher. Refer to [dbt Cloud versions](/docs/dbt-versions/upgrade-core-in-cloud) to upgrade.
+- Avoid using SSH tunneling for [Postgres and Redshift](/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb) connections.
+- Avoid using [PrivateLink](/cloud/secure/about-privatelink).
```
```suggestion
- Avoid using [PrivateLink](/docs/cloud/secure/about-privatelink).
```

mirnawong1 committed:
Update website/docs/docs/cloud/cloud-cli-installation.md

mirnawong1 committed:
Merge branch 'current' into mwong-add-cloud-cli-loom
